libs/template/presets/src/features/b2c/experience/header.ts

```diff
@@ -65,9 +65,8 @@ export const HeaderTemplate: StaticComponent = {
           },
         },
         { type: 'oryx-user-summary' },
-        { type: 'auth-logout' },
```

Comment:
is it deliberate?

Reply:
We are using `oryx-auth-logout` now. Also noticed that `oryx-user-summary` includes the logout component anyways, so auth-logout is not needed. I asked @tobi-or-not-tobi about it earlier.